### ISSUE TITLE
[sunu] Menge Gemüse pro Abholstation

### ISSUE THREAD

**Issue by y0va** [OpenOlitor/OpenOlitor#17]
In unserer Solawi Praxis ist es wichtig zu wissen wieviel von welchem Gemüse an welche Abholstation geliefert werden muss. 
Die bisherigen Lieferscheine sind Korb-basiert. Es wäre gut wenn in den Daten zu der 'Korbübersicht' auch der Inhalt der jeweiligen Körbe mitgeliefert werden würde, bzw die Summen der Gemüsemengen  gleich beim Lieferschein mit eingefügt werden würden.

Oder gibt es dazu bereits eine andere Lösung?

**Comment by AlwinEgger** [comment]
Nein, bisher haben wir im _Korbübersicht_ Report keine Summierung der Gemüsemengen oder ähnliches.
Es ist aber möglich einen entsprechende, personalisierte Auswertung zu erstellen (Administration>Auswertung). Hier könnten beispielsweise die Mengen der Gemüse pro Lieferplanung pro Depot ausgegeben und anschliessend in eine Tabellenkalkulation exportiert werden.

**Comment by y0va** [comment]
Könntest du dafür eine entsprechende SQL Abfrage angeben?

**Comment by ClSchreiber** [comment]
Ich schlage vor, dies (wenn möglich auch) in einem Lieferbericht auszugeben, wir benötigen diesen Lieferbericht in der Schweiz auch.

**Comment by AlwinEgger** [comment]
Eine Abfrage könnte bspw. so aussehen:

```
select 
  da.depot_id, da.depot_name,  lp.produkt_id, lp.produkt_beschrieb, count(k.id) as koerbe, sum(lp.menge) as menge, lp.einheit, sum(lp.preis) as preis
from 
  Korb k 
    join Lieferposition lp on k.lieferung_id = lp.lieferung_id
    join Lieferung l on lp.lieferung_id = l.id
    join DepotAuslieferung da on k.auslieferung_id = da.id
where l.lieferplanung_id = 104
group by da.depot_id, lp.produkt_id, lp.produkt_beschrieb
order by da.depot_id;
```

**Comment by y0va** [comment]
Danke! Für einen täglichen Einsatz ist das so aber zu komplex.

**Comment by AlwinEgger** [comment]
Du hast gesehen, dass unter Administration->Auswertungen Queries abgelegt werden können? In diesem Fall müsste dann jeweils noch die ID der Lieferplanung angepasst werden. Das ist meiner Meinung aber durchaus machbar.

Auf unserer Wishlist hier gibt es die Idee die Auswertungen zu erweitern, so dass zu den Variablen eine Auswahl oder wenigstens ein Formularfeld angezeigt werden.

Grundsätzlich wäre es aber auch möglich das UI für eine oder mehrere Solawis zu erweitern und die so abgelegten Auswertungen als Übersicht als eigenen Menüpunkt anzuzeigen. Da würde keine Backend Anpassung notwendig sein.
